### ISSUE TITLE
상품판매 누적금액이 안 올라갑니다.

### ISSUE THREAD

**Issue by LIMJK** [bjrambo/nurigo#113]
![default](https://cloud.githubusercontent.com/assets/24222320/20832776/8532515e-b8cf-11e6-868f-00886d04ba48.jpg)
위와 같이 판매된 상품 금액이 누적 표시 안됩니다.
판매완료되야 누적되는지요?

**Comment by bjrambo** [comment]
네 아무래도 입금은 입금받은 내용으로만 되어있는 상태이고 실제로 판매가 완료되어 집계가 이루어지는것이 맞는것 같습니다.
